src/main/java/jimmy/mcgymmy/logic/parser/ParserUtil.java

```diff
@@ -30,7 +30,7 @@
  * Contains utility methods used for parsing strings in the various *Parser classes.
  */
 public class ParserUtil {
-
+    private static final String VALIDATION_REGEX = "[0-9]+";
```

Comment:
As this is only change used for the index, is it better to store this within index instead of ParserUtil?

Reply:
ok, I will move it into parseIndex